pytorch_lightning/core/lightning.py

```diff
@@ -1738,7 +1747,8 @@ def save_hyperparameters(
         self,
         *args,
         ignore: Optional[Union[Sequence[str], str]] = None,
-        frame: Optional[types.FrameType] = None
+        frame: Optional[types.FrameType] = None,
+        log: bool = True
```

Comment:
Let's be consistent and use `logger: bool = True`

Reply:
Done!